content/07.pathogenesis.md

```diff
@@ -9,6 +9,11 @@ This section would also be great for the introduction of zoonotic diseases which
 [How does SARS-CoV-2 enter human cells?]
 [What cells are primary infection sites for SARS-CoV-2?]
 [What structural aspects allow for viral entry]?
+ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145;@doi:10.1007/s00134-020-05985-9;doi:10.1128/JVI.00127-20]. 
+The virus invades host cells by using ACE2 for entry and the serine protease TMPRSS2 for S protein priming. 
```

Comment:
Are you asking physically how SARS-CoV-2 can enter through hACE2? I think the receptor-binding domain (RBD) is able to tightly complex with hACE2 when the RBD is in an "up" conformation. The important binding residues are described here:[@doi:10.1038/s41586-020-2179-y] [@doi:10.1101/2020.02.19.956235]. Basically, of the 14 binding residues on the SARS-CoV S protein, 8 of them are conserved in SARS-CoV-2 and the other 6 are semiconservatively substituted. Proteolytic priming (by TMPRSS2, cathepsin B/L, or other proteases) of the S protein is essential, as it allows the S1 subunit to be shed, making way for the S2 subunit to fuse with the host cell membrane. Without these priming cleavage events, the virus cannot fuse with the host cell membrane. I tried to describe this process in detail in my PR (#203), but if it's unclear or incomplete please let me know!!

As for why tigers may be more susceptible, there are two possibilities I can think of. One is that tiger ACE2 may contain residues that allow for stronger interactions with SARS-CoV-2 S protein, allowing for tighter binding and more virulence. Alternatively, tigers may express more proteases (like TMPRSS2, catB/L, furin, etc.) that enhance S protein priming. A paper came out in Science a few days ago showing that SARS-CoV-2 infects cats quite well, whereas it doesn't infect other animals like dogs, pigs, chickens, etc. [@doi:10.1126/science.abb7015]. They don't discuss mechanism, but since tigers are similar to cats it would make sense that tigers could be more susceptible to infection!

Reply:
Thank you. @rando2  Sorry. I didn't go very much in detail.  @anskelly explained it very well.

Reply:
Hi @anskelly that is super interesting! Thank you so much for the links!